docs/DeveloperGuide.md

```diff
@@ -230,14 +230,35 @@ _{Explain here how the data archiving feature will be implemented}_
 
 Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unlikely to have) - `*`
 
-| Priority | As a …​                                    | I want to …​                     | So that I can…​                                                        |
-| -------- | ------------------------------------------ | ------------------------------ | ---------------------------------------------------------------------- |
-| `* * *`  | new user                                   | see usage instructions         | refer to instructions when I forget how to use the App                 |
-| `* * *`  | user                                       | add a new person               |                                                                        |
-| `* * *`  | user                                       | delete a person                | remove entries that I no longer need                                   |
-| `* * *`  | user                                       | find a person by name          | locate details of persons without having to go through the entire list |
-| `* *`    | user                                       | hide private contact details   | minimize chance of someone else seeing them by accident                |
-| `*`      | user with many persons in the address book | sort persons by name           | locate a person easily                                                 |
+| Priority | As a …​                                 | I want to …​                                              | So that I can…​                                                               |
+| -------- | ------------------------------------------ | ------------------------------------------------------------ | -------------------------------------------------------------------------------- |
+| `* * *`  | user                                       | use the application offline                                  | I do not need an internet connection to view my contacts                         |
```

Comment:
The user stories feel out of place. For example this line " As a user, I want to user the application offline, so that I can I do not need an internet connection to view my contacts". 
Perhaps "so that I can view my contacts without an internet connection" would sound better